<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
    <title>Notes: Xiong - 2011 - Bayesian prediction of tissue-regulated splicing using RNA sequence and cellular context - Bodong Chen</title>
    <meta property="og:title" content="Notes: Xiong - 2011 - Bayesian prediction of tissue-regulated splicing using RNA sequence and cellular context - Bodong Chen">
    

    
      
    

    

    
    

    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    <link rel="stylesheet" href="/css/custom.css" />

  </head>

  
  <body class="notes">
    <header class="masthead">
      <h1><a href="/">Bodong Chen</a></h1>

<p class="tagline">Crisscross Landscapes</p>

      <nav class="menu">
        <input id="menu-check" type="checkbox" />
        <label id="menu-label" for="menu-check" class="unselectable">
          <span class="icon close-icon">✕</span>
          <span class="icon open-icon">☰</span>
          <span class="text">Menu</span>
        </label>
        <ul>
        
        
        <li><a href="/">Home</a></li>
        
        <li><a href="/blog/">Blog</a></li>
        
        <li><a href="/cv/">Publications</a></li>
        
        <li><a href="/notes/">Open Scholar</a></li>
        
        <li><a href="/teaching/">Teaching</a></li>
        
        <li><a href="/cn/">博客</a></li>
        
        <li><a href="/index.xml">Subscribe</a></li>
        
        
        </ul>
      </nav>
    </header>

    <article class="main">
      <header class="title">
      
<h1>Notes: Xiong - 2011 - Bayesian prediction of tissue-regulated splicing using RNA sequence and cellular context</h1>

<h3>
  0001-01-01</h3>
<hr>


      </header>





<h2 id="references">References</h2>

<p><strong>Citekey</strong>: @Xiong15092011</p>

<p>Xiong HY, Barash Y and Frey BJ (2011). “Bayesian prediction of
tissue-regulated splicing using RNA sequence and cellular context.”
<em>Bioinformatics</em>, <em>27</em>(18), pp. 2554-2562. <URL:
http://dx.doi.org/10.1093/bioinformatics/btr444>,
<a href="http://bioinformatics.oxfordjournals.org/content/27/18/2554.full.pdf+html">http://bioinformatics.oxfordjournals.org/content/27/18/2554.full.pdf+html</a>,
<URL:
http://bioinformatics.oxfordjournals.org/content/27/18/2554.abstract>.</p>

<h2 id="notes">Notes</h2>

<p>The Bayesian Neural Network technique created / reported in this article represents one of the new advances in neural network, which has some momentum in recent years. It would be interesting to test it in the educational setting, to compare its performance with techniques (like SVM) widely used in educational data mining.</p>

<h2 id="highlights">Highlights</h2>

<p>An important goal in understanding this phenomenon is to infer a ‘splicing code’ that predicts how splicing is regulated in different cell types by features derived from RNA, DNA and epigenetic modifiers. (p. 2554)</p>

<p>Methods: We formulate the assembly of a splicing code as a problem of statistical inference and introduce a Bayesian method that uses an adaptively selected number of hidden variables to combine subgroups of features into a network, allows different tissues to share feature subgroups and uses a Gibbs sampler to hedge predictions and ascertain the statistical significance of identified features. (p. 2554)</p>

<p>Taking a predictive modeling approach, we seek a network model that can be applied to a comprehensive set of RNA sequences to make accurate probabilistic predictions for how splicing will occur under different cellular conditions, using features derived from RNA, DNA, epigenetic modifiers, etc. (p. 2555)</p>

<p>2 LEARNING REGULATORY MODELS (p. 2555)</p>

<p>The set of RNA sequences was obtained by mining EST and cDNA libraries and identifying 3665 cassette-type mouse exons, i.e. exons that are sometimes included and sometimes excluded in the spliced transcripts extracted from tissues and cell lines. The fraction of isoforms including each cassette exon was profiled across 27 mouse tissues. (p. 2555)</p>

<p>We are not really interested in exactly how only a single transcript is spliced at a particular point in time within a particular cell. This knowledge would neither provide an understanding of how splicing works nor enable us to predict what would happen under different conditions. A more compelling scientific goal is to infer a network model that summarizes related splicing patterns, accounts for variability in cellular conditions and how they influence splicing (p. 2555)</p>

<p>Measuring code quality using relative log-likelihood (p. 2556)</p>

<p>The feature vectors form a 3665×1014 matrix, with rows corresponding to exons and columns corresponding to features, and the tissue-dependent targets form a 3665×12 matrix, with each row containing qinc, qexc and qnc for each of the four tissue types. (p. 2556)</p>

<p>Training sets and test sets: (p. 2556)</p>

<p>The number of features is large relative to the number of examples, so methods that are not regularized will likely overfit the data. (p. 2556)</p>

<p>Care needs to be taken to avoid overfitting the model in such a way that generalization is not possible. (p. 2556)</p>

<p>3 ALGORITHMS (p. 2556)</p>

<p>3.1 Bayesian Neural Network (p. 2556)</p>

<p>To address these issues, we take a Bayesian approach (Bishop, 2006; MacKay, 1992; Neal, 1996) where the computational task is to sample from a posterior distribution over models. Predictions for novel test cases are made by averaging the predictions from the sample of models, and important features can be identified by checking to see if they are used more frequently than expected at random under the prior distribution of models. (p. 2557)</p>

<p>3.2 Other methods included in the benchmark (p. 2557)</p>

<p>4 RESULTS (p. 2558)</p>

<p>Comparisons of test code quality: All methods were evaluated using held out test data as described above and the test code quality for several methods is plotted in Figure 1 (see below for further details). (p. 2558)</p>

<p>The Bayesian method achieved a relative improvement of 52% over the previously published result obtained using boosting (Barash et al., 2010a), and significantly outperforms the other methods. (p. 2558)</p>

<p>Comparisons of classification accuracy: The log-likelihood based measure of code quality described above takes into account how accurately each method can assess its confidence in its predictions. A different, but related task is to apply a threshold to each method’s prediction probabilities and measure classification accuracy. (p. 2558)</p>


  <footer>
  
<nav class="post-nav">
  <span class="nav-prev">&larr; <a href="/notes/2017-07-10-yamagata-lynch2010-eu/">Notes: Yamagata-Lynch, L. C. (2010). Understanding Cultural Historical Activity Theory</a></span>
  <span class="nav-next"><a href="/notes/2015-08-13-worsley2012/">Notes: Worsley - 2012 - Multimodal learning analytics</a> &rarr;</span>
</nav>



<section class="comments">
  <div id="disqus_thread"></div>
  <script>
  var disqus_config = function () {
  
  };
  (function() {
    var inIFrame = function() {
      var iframe = true;
      try { iframe = window.self !== window.top; } catch (e) {}
      return iframe;
    };
    if (inIFrame()) return;
    var disqus_js = '//meefen.disqus.com/embed.js';
    var d = document, s = d.createElement('script');
    s.src = disqus_js; s.async = true;
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
  })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</section>



<script src="//yihui.name/js/math-code.js"></script>
<script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML"></script>

<script async src="//yihui.name/js/center-img.js"></script>

  

  
  <hr>
  <div class="copyright">&copy; <a href="http://bodong.ch/">Bodong Chen</a> 2018 | <a href="https://github.com/meefen">Github</a> | <a href="https://twitter.com/bod0ng">Twitter</a></div>
  
  </footer>
  </article>
  
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-50108133-1', 'auto');
ga('send', 'pageview');
</script>

  </body>
</html>

